stable/heapster/templates/deployment.yaml

```diff
@@ -14,6 +14,9 @@ spec:
       labels:
         app: {{ template "fullname" . }}
     spec:
+{{- if .Values.rbac.create }}
+      serviceAccountName: {{ template "fullname" . }}
```

Comment:
Create an `rbac.serviceAccountName` and use `serviceAccountName: {{ if .Values.rbac.create }}{{ template "fullname" . }}{{ else }}"{{ .Values.rbac.serviceAccountName }}"{{ end }}`